Clicker/ViewControllers/PollBuilderViewController.swift

```diff
@@ -410,14 +411,16 @@ class PollBuilderViewController: UIViewController {
         dismiss(animated: true, completion: nil)
         hideKeyboard()
         
-        switch questionType {
+        switch questionType { 
```

Comment:
make sure you have trim trailing whitespaces enabled!
![](https://pbs.twimg.com/media/DTawMEMWsAAbbbs.png)